Sources/PrimerSDK/Classes/Error Handler/PrimerError.swift

```diff
@@ -384,11 +384,24 @@ public enum PrimerError: PrimerErrorProtocol {
         case .missingPrimerDelegate:
             return "Primer's delegate has not been set. Ensure that you have added Primer.shared.delegate = self on the view controller you wish to present Primer's SDK."
         case .missingPrimerCheckoutComponentsDelegate:
-            return "Primer Checkout Components' delegate has not been set. Ensure that you have added PrimerCheckoutComponents.delegate = self on the view controller you wish to implement the components."
+            let message =
+"""
+Primer Checkout Components' delegate has not been set.
```

Comment:
You can put `\` at the end of these lines to preserve the way they are sent to the console + backend.

Reply:
Not sure I understand completely...

Reply:
The new multi-line string will add `\n` in at the end of each line. If you add `\` as the last character it won't do that, which will preserve the format of the original string.